### PR TITLE
Model eval

### PR DESCRIPTION
**Pull request by joris68**
- added a serialized mock model
- added evaluation metrics in the EvalController Path:
      1. recall
      2. precision
      3. f1 Score
 - build out the model
 - build out a custom dataset with the pyTorch api for stochastic gradient descent

### CODE REVIEW

**Review by LHMoritz — APPROVED** [review]
lgtm
